title-2-conservation/eqip_ira/eqip_ira_state_distribution.json

```diff
@@ -0,0 +1,16975 @@
+[
+    {
+        "state": "TX",
+        "practices": [
+            {
+                "practiceName": "Total",
```

Comment:
Quick question, I was referring our current title II setup
<img width="797" alt="Screenshot 2024-05-17 at 3 40 40 PM" src="https://github.com/policy-design-lab/data-import/assets/92752107/98517010-9c1d-4d45-92fa-fcbe5d69bafc">
We make the totalPayment as the same level as the state name. Should we follow the same pattern?

Reply:
It only has 2023 data and future data is not complete. Some of them has values and some doesn't hmmm maybe we want discuss about that. Also there is no statute either. All of them are EQIP IRA program and it has practices under it.

Reply:
Sorry if I miscommunicated. For IRA, we have a `"practiceName": "Total"... `JSON item, while in the regular data, we just use `totalPaymentInDollars` instead. Can we do the same? Or did you mean that not all practices have a total?

It's okay that we don't have 'status'; I will look for 'practices' instead, as you have already set up.

Reply:
There is a state name as 'total' in the table and I don't know what it means exactly and my assumption that is tit is the total for all the state. I strongly suggest you to check the 20240215_EQIP_IRA.csv file

Reply:
Interesting...@sandeep-ps do you have any clue about the 'total' means? Should we seek clarification with Johnathan?

Reply:
Sorry, I missed this message/email notification earlier. I think this state name with the value "Total" can be ignored, as we can calculate the total while generating the eqip_ira_summary.json. If possible, please include `totalPracticeInstanceCount` and `totalPaymentInDollars` at the root level of eqip_ira_summary.json.

We also have a practice name with the value "Total", which needs to be clarified with Yifan. I will send an email and CC you both. Thanks.

Reply:
Based on feedback from Yifan via emails (in which you are copied), it looks like we can remove both the state name "Total" and practice name "total" from different state data in the JSON file. 

As I suggested in my previous comment in this thread, if we can include `totalPracticeInstanceCount` and `totalPaymentInDollars` at the root level of eqip_ira_summary.json, it will be sufficient to get a national-level total for dollars and instance counts. Thanks.

Reply:
@sandeep-ps I removed the total from as state name and removed total as practice name. Also, add  totalPracticeInstanceCount and totalPaymentInDollars of national level at the root elve or summary.json

Reply:
Thank you!